test/integration/install-test.js

```diff
@@ -78,6 +80,7 @@ describe(install, function() {
       fixturesPath: 'test/fixtures/blueprint/addon/legacy-app/local/no-addon',
       commitMessage: 'my-app',
       addon: location,
+      blueprintName: 'ember-cli-update-git-addon-test',
```

Comment:
Did we lose the default blueprint test by changing this?

Reply:
Modified code to ensure default blueprint https://github.com/ember-cli/ember-cli-update/pull/1084/commits/8579ff6a9d1b1207e3ec1953e25194556dba79f0 installed if no override is passed 

Also added code to accommodate https://cli.emberjs.com/release/writing-addons/addon-blueprints/#blueprintconfig